integration_tests/configs/ibc.jsonnet

```
@@ -140,7 +140,7 @@ config {
       {
         id: 'cronos_777-1',
         max_gas: 500000,
-        gas_multiplier: 2,
+        gas_multiplier: 10,
```

seems a bit too large
```suggestion
        gas_multiplier: 4,
```